alf/trainers/policy_trainer.py

```diff
@@ -351,32 +348,15 @@ def __init__(self, config: TrainerConfig):
             self._eval_summary_writer = alf.summary.create_summary_writer(
                 self._eval_dir, flush_secs=config.summaries_flush_secs)
 
-    @gin.configurable('alf.trainers.RLTrainer._create_environment')
-    def _create_environment(self,
```

Comment:
Can you also update knowledge_base.rst too? `_create_environment` is mentioned there.

Reply:
updated